src/common/src/util/value_encoding/column_aware_row_encoding.rs

```diff
@@ -75,13 +75,13 @@ impl RowEncoding {
                 self.flag |= Flag::OFFSET16;
                 usize_offsets
                     .iter()
-                    .for_each(|m| self.offsets.put_u16(*m as u16));
+                    .for_each(|m| self.offsets.put_u16_le(*m as u16));
```

Comment:
cc @Eurekaaw  Could you please add more unit tests for large row?

Reply:
my bad 😭